examples/ima-video-responsive.amp.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
```

Comment:
As-is this implementation doesn't actually support more than one player on the same page. I can modify it to do so, but not by next Wednesday. Is it OK if I just go with one sample for now?